scripts/vm/hypervisor/xenserver/logrotate

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Version @VERSION@
+#
+# script to perform logrotation on xenserver 6.0.2 and later
+
+/usr/sbin/logrotate  /etc/logrotate.d/cloudlog
```

Comment:
@sspans  I think the files mentioned in the patch file are the only ones getting deployed to the xenserver. Of these I cloud find cloudlog is the only logrotate configuration file. 
Please let me know if I missed anything.